### PR TITLE
feat(scss): override bootstrap color them by refactoring scss handling

### PR DESCRIPTION
**Pull request by seadubs**
Refactor scss handling out of webpack, instead use sass-dart to compile. Update package.json with
new commands, refactor old commands ot mak to make usage more c more clear. Fix linting issues.

re: https://github.com/oslabs-beta/infernode/issues/180

### CODE REVIEW

**Comment by 8000meters** [comment]
Why does it have all those old commits?

**Comment by seadubs** [comment]
image

File: src/routes/captures.router.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const captureRouter = Router();
 
 // Create new flamegraph from a .perf
 captureRouter.post(
-  '/flamegraph',
+  '/:flamegraph',
```
Comment (dzidupe) [inline src/routes/captures.router.ts:15]: Did you intentionally add the colons (here and for /api/capture/icicle) back? I think they might cause a bug with the way req.params is handled

**Comment by seadubs** [comment]
> Why does it have all those old commits?

Hmmm, it looks like when I merged dev into my working branch it picked those up. I'll cherry pick my commits off onto a new branch and retry.

**Pull request closed without merging**